students/MikeShand/lesson03/ListLab.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+
+#Series 1
+#Here's the initial list
+
+fruit=['Apples','Pears','Oranges','Peaches']
+print(fruit)
+
+#This prompts the user to add a fruit, then makes a new list and prints the result
+
+n=input('what fruit would you like to add?')
+fruit2=[n]
+fruit3=fruit+fruit2
+print(fruit3)
+
+#Now the user selects a number, and this function displays the associated fruit
+
+x=int(input('what number fruit would you like to see?'))
+if x<=len(fruit3) and x!=0:
+    print(x, fruit3[x-1])
+else:
+    print("Invalid selection")
+
+#This demonstrates adding a user defined fruit with +
+
+y=input('what other fruit would you like to add?')
+fruit4=[y]
+fruit5=fruit4+fruit3
+print(fruit5)
+
+#Now we'll add a user defined fruit with the insert operation
+
+z=input('Come on! Add one more fruit!')
+fruit5.insert(0,z)
+
+#Here we use startswith to print a list of fruits that start with P
+
+for i in fruit5:
+    if i.startswith('P'):
+        print('Here are your P fruits:',i)
+
+#Series 2
+
+print('Now lets start series two')
+print("Here's the same series again")
+print(fruit)
+print("And here it is with the last fruit removed")
+fruit.remove("Peaches")

```

Comment:
let's assume we don't a value of the last fruit to remove.. how would you remove the last fruit by it's position rather than value?